docs/source/include/learning.rst

```diff
@@ -0,0 +1,93 @@
+Learned Models
+==============
+
+In SCICO, neural network models are used to represent imaging problems and provide different modes of data-driven regularization.
+The models are implemented in `FLAX <https://flax.readthedocs.io/>`_, and constitute a representative sample of frequently used networks.
+
+
+FlaxMap
+-------
+
+SCICO interfaces with the implemented models via :class:`.FlaxMap`. This provides a standardized access to all trained models via the model definiton and the learned parameters. Further specialized functionality, such as learned denoisers, are built on top of :class:`.FlaxMap`. The models implemented are described next.
+
+
+
+DnCNN
+-----
+
+The denoiser convolutional neural network model (DnCNN), :class:`.DnCNNNet`, from :cite:`zhang-2017-dncnn` is used to denoise images that have been corrupted with additive Gaussian noise.
+
+
+
+ODP
+---
+
+The unrolled optimization with deep priors (ODP), :class:`.ODPNet`, from :cite:`diamond-2018-odp` is used to solve inverse problems in imaging by adapting classical iterative methods into an end-to-end framework that incorporates deep networks as well as knowledge of the image formation model.
+
+The framework aims to solve the optimization problem
+
+    .. math::
+       \argmin_{\mb{x}} \; f(A \mb{x}, \mb{y}) + r(\mb{x}) \;,
+
+where :math:`A` represents a linear forward model and :math:`r` a regularization function encoding prior information, by unrolling the iterative solution method into a network where each iteration corresponds to a different stage in the ODP network.
+
+
+Unrolled Optimization Algorithms
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Different iterative solution algorithms produce different ODP networks. The ones implemented in SCICO are
+
+* Proximal Map: It corresponds to solving
+
```

Comment:
Suggested: `Proximal Map: This algorithm corresponds to solving`

Reply:
Modified as suggested.